src/components/SelectAssets/SelectAssetView.tsx

```diff
@@ -1,13 +1,6 @@
 import { Asset } from '@shapeshiftoss/types'
 import { useEffect } from 'react'
-import {
-  Redirect,
-  Route,
-  RouteComponentProps,
-  Switch,
-  useHistory,
-  useLocation
-} from 'react-router-dom'
+import { Route, Routes, useLocation, useNavigate } from 'react-router-dom'
```

Comment:
@developerfred - can you take a look at this file, its failing to build in our CI.  I just pulled it down locally to also try and I am getting the following
```
SyntaxError: shapeshift/web/src/components/SelectAssets/SelectAssetView.tsx: 'return' outside of function. (41:2)
  39 |   }, [navigate])}
  40 |   
> 41 |   return (
     |   ^
  42 |   <Routes location={location} key={location.key}>
  43 |     <Route
  44 |       path={SelectAssetRoutes.Search}
```

Reply:
fixed here 8affb97

Reply:
TY!